### DISCUSSION TITLE
PRO account issue

### DISCUSSION THREAD

**Discussion by harlantwood** [mdbootstrap/TW-Elements#2325] [Support from community] [answered]
Hi, not sure where to get support for a paid acount -- I just purchased a license but I still can't see API docs, it logs me in "successfully" it says, but then when I click on the API tab it just asks me to log in or purchase again 😭 

I can't find anywhere to email or otherwise contact support, please help.

**Comment by catchmareck** [comment, marked as answer]
Can you check now? It should be working now

**Comment by harlantwood** [comment]
Yes, works now, thanks 👍
